backend/app/schema/recommendation.py

```diff
@@ -115,6 +115,12 @@ class StructuredQuery(BaseModel):
     trip_type: Optional[TripType] = None
 
 
+class AdditionalRecommendationsQuery(BaseModel):
+    original_query: StructuredQuery
```

Comment:
As we talked in slack and issue, we don't need original data about user preferences anymore as we also don't save them in db, so we can delete `original_query`.
https://github.com/chanon-mike/smart-ryokou/issues/75